### DISCUSSION TITLE
Should I write an installer for Volume Control

### DISCUSSION THREAD

**Discussion by radj307** [radj307/volume-control#52] [Community Polls]
Should I write an installer for Volume Control to automate the setup process for users who don't want/need a portable version of Volume Control?

**Comment by ashenhollow0** [comment]
by any chance have you done this already and i just cant find it? if not would it be possible to make one im so confused.

**Reply by StormingChaos** [reply]
No there is currently no installer. To install volume control just download and launch VolumeControl.exe. If the window does not open, check your system tray to see if it launched silently.

**Comment by radj307** [comment]
An installer is now available as of version 6.0.0
